.gitignore

```diff
@@ -114,6 +114,3 @@ deployment/ansible/*.retry
 ./data
 ./notebooks
```

Comment:
These actually do not seem to work. My understanding is that `/data/` or `data/` will ignore the `data` directory, while `/data` will additionally ignore any file named `data`.

I think `/data/` and `/notebooks/` is what we want.